src/renderer/dx/DxFontInfo.h

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#pragma once
+
+#include <dwrite.h>
+#include <dwrite_1.h>
+#include <dwrite_2.h>
+#include <dwrite_3.h>
+
+namespace Microsoft::Console::Render
+{
+    class DxFontInfo
+    {
+    public:
+        DxFontInfo() noexcept;
+
+        DxFontInfo(std::wstring_view familyName,

```

Comment:
I'm not sure whether this should be `familyName` of `faceName`. Seems to me there are `familyName` & `faceName` in the GDI font info, but only ` familyName` in DWrite?